dlrs/clearlinux/tensorflow/mkl/scripts/install_tensorflow.sh

```diff
@@ -55,50 +54,32 @@ run() {
 python_pkgs(){
 # install dependencies for tensorflow build
   pip install pip six numpy wheel setuptools mock "future>=0.17.1"
-  pip install keras_applications==1.0.6 --no-deps
-  pip install keras_preprocessing==1.0.5 --no-deps
+  pip install keras_applications==1.0.8 --no-deps
+  pip install keras_preprocessing==1.1.0 --no-deps
 }
 
 get_project() {
-  git clone https://github.com/${PROJECT}/${PROJECT}.git 
+  git clone --depth 1 -b ${TF_TAG} https://github.com/${PROJECT}/${PROJECT}.git 
   cd tensorflow && git checkout ${TF_TAG}
   apply_patches
 }
 
 apply_patches() {
   git config --global user.email "example@example.com"
   git config --global user.name "example@example.com"
+  # fix from https://github.com/Meadosc/patches/blob/master/0001-change-reserved-tp_print-slot-from-nullptr-to-0.-Fix.patch
   wget https://raw.githubusercontent.com/clearlinux-pkgs/tensorflow/master/0001-Add-grpc-fix-for-gettid.patch 
   git am 0001-Add-grpc-fix-for-gettid.patch
-
+  # fix from https://github.com/Meadosc/patches/blob/master/0001-change-reserved-tp_print-slot-from-nullptr-to-0.-Fix.patch
+  wget https://raw.githubusercontent.com/Meadosc/patches/master/0001-change-reserved-tp_print-slot-from-nullptr-to-0.-Fix.patch
+  git am 0001-change-reserved-tp_print-slot-from-nullptr-to-0.-Fix.patch
+  # fix from https://github.com/clearlinux-pkgs/tensorflow/blob/master/0001-Fix-TensorFlow-on-Python-3.8-logger-issue.patch
+  wget https://raw.githubusercontent.com/clearlinux-pkgs/tensorflow/master/0001-Fix-TensorFlow-on-Python-3.8-logger-issue.patch
+  git am 0001-Fix-TensorFlow-on-Python-3.8-logger-issue.patch
 }
 
 build () {
-  # configure and build tensorflow avx2 and avx512 versions
-  # build avx2 TF
-  mkdir -p /tmp/tf/avx2
-  export ARCH=skylake
-  export TUNE=skylake
-  export TF_BUILD_MAVX=MAVX2
-  export CC_OPT_FLAGS="-march=${ARCH} -mtune=${TUNE}"
-  export CFLAGS="$CFLAGS -march=${ARCH}"
-  export CXXFLAGS="$CXXFLAGS -march=${ARCH}"
-
-  #skl instructions
-  #--copt=-mavx2
-  # --copt="-DEIGEN_USE_MKL_VML" 
-  ./configure
-  bazel --output_base=/tmp/bazel build  \
```

Comment:
I would prefer us commenting this section rather than removing it

Reply:
Is that for historical reference in case we go back to bazel?

Reply:
nop, we dont want that, let's  just the env variables. line 77 to 85, what do you think?

Reply:
added back in as commented lines below avx512 install